sormas-e2e-tests/src/main/java/org/sormas/e2etests/pojo/Contact.java

```diff
@@ -51,4 +51,5 @@
   String contactCategory;
   String relationshipWithCase;
   String descriptionOfHowContactTookPlace;
+  String contactId;
```

Comment:
Please rename it to `uuid`, because you are already in `Contact` class

Reply:
will do. but it's not an UUID. i've tried to make it an UUID object but it's failing the conversion.

Reply:
@BojteTamas -> updated

Reply:
it's UUID, but it is created in other way than `UUID.random().toString()`